clients/js/cmds/sui.ts

```diff
@@ -37,6 +63,28 @@ exports.command = 'sui';
 exports.desc = 'Sui utilities ';
 exports.builder = function(y: typeof yargs) {
   return y
+    .command("get-owned-objects", "Get owned objects by owner", (yargs)=>{
+      return yargs
+        .option("network", network_options)
+        .option("rpc", rpc_description)
+        .option("owner", {
+          alias: "o",
+          describe: "Owner address",
+          required: true,
+          type: "string",
+        })
+    }, async (argv)=>{
+      const network = argv.network.toUpperCase()
+      const rpc = argv.rpc ?? NETWORKS[network]["sui"].rpc;
+      const provider = new JsonRpcProvider(rpc);
+      const owner = argv.owner;
+      const objects = await provider.getObjectsOwnedByAddress(
+        owner
+      );
+      console.log("network: ", network)
+      console.log("owner: ", owner)
+      console.log(objects)
```

Comment:
Should this be JSON stringified so we can read nested objects?

Reply:
stringifying it displays all nested layers? I didn't know that :O